.circleci/config.yml

```diff
@@ -13,7 +13,12 @@ jobs:
       # -batch is necessary to make sure sbt doesn't stall waiting for user input in case of certain failures
       - run: echo '-batch' >> .sbtopts
       - run: sbt update
-      - run: sbt +scalafmtCheckAll
+      - run:
+          name: Checking formating
+          command: sbt "scalafmtCheckAll"
+      - run:
+          name: Checking import order
```

Comment:
Also, add to README how to build the project + that `sbt scalafixAll` should be run before a push (could be `pre-commit` git hook theoretically... but if such hooks are becoming too heavy, people tend to turn them off ;p)

Reply:
`sbt scalafixAll` is a heavy operation, as it compiles code from zero

Reply:
Hmmm okay but:
1. doesn't it just use the output of `sbt compile`? as in, does it really **recompile** everything from scratch even if the code has not been change since the last compilation? I've never observed such a full recompilation in Hydra
2. is there any alternative way to ensure all code conforms to scalafix rules?

Reply:
Yeah, it takes around 30 seconds to complete a hook